### PR TITLE
kde-base/kdelibs: Version bump to 4.14.13

### PR DESCRIPTION
**Pull request by a17r**
(ebuild based on kdelibs-4.9999 from kde overlay)

### CODE REVIEW

**Comment by mgorny** [comment]
@gentoo/kde

**Comment by mgorny** [comment]
@kensington, we could do some usex here but shhh.

**Comment by kensington** [comment]
Thanks, applied.

@mgorny, that's the biggest problem you have with kdelibs ebuild!?

File: kde-base/kdelibs/kdelibs-4.14.13.ebuild [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+	kde4-base_src_configure
+}
+
+src_compile() {
+	kde4-base_src_compile
+
+	# The building of apidox is not managed anymore by the build system
+	if use doc; then
+		einfo "Building API documentation"
+		cd "${S}"/doc/api/
```
Comment (hasufell) [inline kde-base/kdelibs/kdelibs-4.14.13.ebuild]: missing `|| die`

File: kde-base/kdelibs/kdelibs-4.14.13.ebuild [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+src_install() {
+	kde4-base_src_install
+
+	# use system certificates
+	rm -f "${ED}"/usr/share/apps/kssl/ca-bundle.crt || die
+	dosym /etc/ssl/certs/ca-certificates.crt /usr/share/apps/kssl/ca-bundle.crt
+
+	if use doc; then
+		einfo "Installing API documentation. This could take a bit of time."
+		cd "${S}"/doc/api/
```
Comment (hasufell) [inline kde-base/kdelibs/kdelibs-4.14.13.ebuild]: missing `|| die`

**Pull request closed without merging**